db/institutions.yml

```diff
@@ -938,6 +995,43 @@ unisa:
   name: University of Salerno
   state: ''
   zip: '84084'
+uofdayton:
+  aka: []
+  city: Dayton
+  country: USA
+  day: 10
+  month: 1
+  name: University of Dayton
+  state: Ohio
+  updated: '2020-01-10 17:12:54.190176'
```

Comment:
I think I fixed the validator so that it will pass if this is a datetime object, so please can you remove everywhere the quotes on these ``updated`` entries?  Sorry about that!